src/Jellyfin.LiveTv/Listings/SchedulesDirect.cs

```diff
@@ -613,6 +613,7 @@ private async Task<string> GetToken(ListingsProviderInfo info, CancellationToken
 
             // Response is automatically disposed in the calling function,
             // so dispose manually if not returning.
+#pragma warning disable IDISP016, IDISP017
```

Comment:
This function is broken BTW
https://github.com/jellyfin/jellyfin/issues/8502#issuecomment-1341123200